### DISCUSSION TITLE
Add wrapping() method to Field class

### DISCUSSION THREAD

**Discussion by phattarachai** [laravel/nova-issues#5525] [Q&A] [answered]
Hi,

Currently, if I have a long message in a text field, I need to wrap it up inside the table cell using the following code:

```php
Text::make('messages')
    ->withMeta([
        'wrapping' => true
    ]),
```

While this approach works, it feels a bit cumbersome to me. I would prefer to have a simpler and more elegant way to achieve this, such as:

```php
Text::make('messages')
    ->wrapping(),
```

I have searched the Nova documentation but could not find any information about wrapping text. Therefore, I am wondering if a similar method already exists or if this feature could be added to Nova.

Thank you.

**Comment by crynobone** [comment, marked as answer]
You can add macros to `Field`.

```php
Field::macro('wrapping', function () {
  // ....

  return $this;
});
```

https://tighten.com/insights/the-magic-of-laravel-macros/
